pkg/rtc/mediatracksubscriptions.go

```diff
@@ -0,0 +1,499 @@
+package rtc
```

Comment:
The idea is to be able to use this as is in the remote media track too.

Reply:
Good idea, Thanks